packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -562,4 +576,12 @@ export class BlobManager {
             this.redirectTable.set(storageId, storageId);
         }
     }
+
+    public getPendingBlobs(): IPendingBlobs {
+        const blobs = {};
+        for (const [key, entry] of this.pendingBlobs) {
+            blobs[key] = { blob: bufferToString(entry.blob, "utf8") };
```

Comment:
Is it safe to interpret arbitrary blobs as utf8?  My understanding is that certain byte sequences are illegal in UTF-8, but possibly the conversion between buffer/string does not enforce well-formedness and round-trips?

Reply:
Good catch